src/test/aws/construct/static-asset-deployment.test.ts

```diff
@@ -1,17 +1,17 @@
 import * as cdk from 'aws-cdk-lib'
+import { App, StackProps } from 'aws-cdk-lib'
 import { Template } from 'aws-cdk-lib/assertions'
 import * as s3deploy from 'aws-cdk-lib/aws-s3-deployment'
 import { Construct } from 'constructs'
-import { CommonStack, StaticAssetDeployment, StaticAssetDeploymentProps } from '../../../lib'
+import {
+  CommonStack,
+  StaticAssetDeployment,
+  StaticAssetDeploymentProps,
+  CommonConstruct,
+  CommonStackProps,
+} from '../../../lib'
 
-interface TestStackProps extends StaticAssetDeploymentProps {
-  staticAssetBucket: any
-  staticAssetDeployment: any
-  staticAssetSources: any
-  staticAssetsForExport: any
-}
-
-const testStackProps = {
+const testContext = {
```

Comment:
separate between what comes from context and what comes from properties, before all was munged to single object

IMO it is good to keep that separation to understand what comes from context and what should come for properties